test/decorators/user_decorator_test.rb

```diff
@@ -69,4 +70,37 @@ class UserDecoratorTest < ActiveDecoratorTestCase
     assert_equal @admin_mentor_user.editor_or_other_editor, 'textbringer'
     assert_equal @student_user.editor_or_other_editor, 'VSCode'
   end
+
+  test '#completed_fraction don\'t calculate practice that include_progress: false' do
+    user = @admin_mentor_user
+    old_fraction = user.completed_practices_include_progress.size
+    user.completed_practices << practices(:practice5)
+
+    assert_not_equal old_fraction, user.completed_fraction
+
+    old_fraction = user.completed_practices_include_progress.size
+    user.completed_practices << practices(:practice53)
+
+    assert_equal old_fraction, user.completed_practices_include_progress.size
+  end
+
+  test '#completed_fraction don\'t calculate practice unrelated cource' do
+    old_fraction = @admin_mentor_user.completed_practices_include_progress.size
+    @admin_mentor_user.completed_practices << practices(:practice5)
+
+    assert_not_equal old_fraction, @admin_mentor_user.completed_practices_include_progress.size
+
+    old_fraction = @admin_mentor_user.completed_practices_include_progress.size
+    @admin_mentor_user.completed_practices << practices(:practice55)
+
+    assert_equal old_fraction, @admin_mentor_user.completed_practices_include_progress.size
+  end
+
+  test '#completed_fraction_in_metas' do
+    fraction_in_metas = '2 （必須:1）'
+    @non_required_subject_completed_user.completed_practices << practices(:practice5)
+    @non_required_subject_completed_user.completed_practices << practices(:practice61)
+
+    assert_equal @non_required_subject_completed_user.completed_fraction_in_metas, fraction_in_metas
```

Comment:
```suggestion
    assert_equal fraction_in_metas, @non_required_subject_completed_user.completed_fraction_in_metas
```

テスト内容は良いと思いました🙆

細かい点ですが、`assert_equal` メソッドを使用する際には、第一引数に「期待する値」を、第二引数に「テスト対象の値」を置くのが一般的と、上記のようにするのはどうでしょうか？

以前、私もどこかのプラクティスで教えていただいたので共有となります🙏

[参考：assertEqualsやassert_equalの引数はなぜ expected, actual の順なのか、調べてみた](https://qiita.com/jnchito/items/a8a16a8242889b41d8ec#:~:text=2%E3%81%A4%E3%81%AE%E5%80%A4%E3%81%8C%E7%AD%89%E3%81%97%E3%81%84%E3%81%93%E3%81%A8%E3%82%92%E6%A4%9C%E8%A8%BC%E3%81%99%E3%82%8B%E3%83%A1%E3%82%BD%E3%83%83%E3%83%89%EF%BC%88assertEquals%E3%82%84assert_equal%E3%81%AA%E3%81%A9%EF%BC%89%E3%81%AE%E5%BC%95%E6%95%B0%E3%81%8C%20expected%2C%20actual%EF%BC%88%E6%9C%9F%E5%BE%85%E3%81%99%E3%82%8B%E5%80%A4%E3%80%81%E5%AE%9F%E9%9A%9B%E3%81%AE%E5%80%A4%EF%BC%89%E3%81%AE%E9%A0%86%E3%81%AB%E4%B8%A6%E3%81%B3%E3%81%BE%E3%81%99%E3%80%82)

Reply:
了解しました！変更しました！